src/viewer.js

```diff
@@ -531,6 +532,12 @@ ReactSVGPanZoom.propTypes = {
   //Turn off zoom on double click
   disableDoubleClickZoomWithToolAuto: PropTypes.bool,
 
+  //toolbar props
+  toolbarProps: PropTypes.shape({
```

Comment:
I didn't restrict the shape of `toolbarProps` here in order not to restrict props for custom toolbars.
But it's surely safer this way :)